openapi/src/main/scala/endpoints/delegate/DocumentedRequests.scala

```diff
@@ -0,0 +1,34 @@
+package endpoints
+package delegate
```

Comment:
Oh, I have missed that both algebra and interpreter is in the same project. Agree, it is a good idea, but I wonder if we need to separate algebra and interpreter in such case. If connected, one would defined documentation in common project and then mix in delegates in client/server(not sure if that is possible to implement, just throwing ideas).